### ISSUE TITLE
Not slicing correctly 

### ISSUE THREAD

**Issue by vic014** [Ultimaker/Cura#12757]
### Application Version

4.13.1

### Platform

Windows 11

### Printer

CR10s

### Reproduction steps

Opened file in Cura, went to preview, this is what i see.
![09D8BBF5-9B68-4E19-B31F-4D027E486E74](https://user-images.githubusercontent.com/40169987/178842278-241b1a9b-e6c6-4be2-bfc1-c60f79788ac0.jpeg)

### Actual results

Not sliced properly 

### Expected results

Sliced object ready for printing 

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

Nothing else

**Comment by Asterchades** [comment]
Without a log or project I can only guess, but it would appear that you've either told it not to display the wall lines (the settings would be just above where the provided image stops at the top) or you've told it that the wall width and skin thickness are 0mm.

I would hazard a guess that the "Shell" line type has simply been disabled in this case. But you will need to provide more information should that not be the case.

**Comment by Ghostkeeper** [comment]
Yeah, the pictures are identical. We'll close this. It was a user error, having accidentally disabled the visualisation of the shell.

**Comment by vic014** [comment]
Yes it can

Thank You

Victor Lawver

On Jul 14, 2022, at 08:54, GregValiant ***@***.***> wrote:

﻿

I saw that same image on Reddit<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.reddit.com%2Fr%2FCura%2Fcomments%2Fvye5zg%2Fthis_a_glitch_in_the_matrix_or_a_wrong_setting%2F&data=05%7C01%7C%7C989b92cb7b044e63209e08da6597f43f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637934000566747422%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=8Z6Wj8pOWaH66rtHuWDYoU7362ukD721RmzBy7zguLM%3D&reserved=0> and it appears that this was sorted out by enabling the Shell linetype? Can this be closed?
